analysis/figures/plot_cohorts.R

```diff
@@ -0,0 +1,123 @@
+                           model = character(),
+                           estimate = numeric(),
+                           conf.low = numeric(),
+                           conf.high = numeric(),
+                           cohort = character())
+    
+  }
+  
+  # Make single dataset for plotting ---------------------------------------------
+  
+  df <- rbind(df_prevax, df_vax, df_unvax)
+  
+  # Handle time ------------------------------------------------------------------
+  
+  df$start <- as.numeric(gsub("days","",sub('\\_.*', '', df$term)))
+  df$stop <- as.numeric(gsub(".*_", "",df$term))
+  df$time <- df$start + ((df$stop-df$start)/2)
+  
+  max_weeks <- max(df$stop)/7
+  
+  # Plot -------------------------------------------------------------------------
+  
+  ggplot2::ggplot(data=df, 
+                  mapping = ggplot2::aes(x=time/7, y = estimate, color = model, fill=model)) + 
+    ggplot2::geom_point(position = ggplot2::position_dodge(width = 1)) +
+    ggplot2::geom_hline(mapping = ggplot2::aes(yintercept = 1), colour = "#A9A9A9") +
+    ggplot2::geom_errorbar(mapping = ggplot2::aes(ymin = ifelse(conf.low<0.25,0.25,conf.low), 
+                                                  ymax = ifelse(conf.high>64,64,conf.high),  
+                                                  width = 0), 
+                           position = ggplot2::position_dodge(width = 1)) +   
+    ggplot2::geom_line(position = ggplot2::position_dodge(width = 1)) +    
+    ggplot2::scale_y_continuous(lim = c(0.5,32), breaks = c(0.5,1,2,4,8,16,32), trans = "log") +
+    ggplot2::scale_x_continuous(lim = c(0,max_weeks), breaks = seq(0,400,4)) +
+    ggplot2::scale_fill_manual(values = c("#bababa","#000000"), 
+                               breaks = c("mdl_agesex","mdl_max_adj"), 
+                               labels = c("Age/sex adjustment","Extensive adjusted"))+ 
+    ggplot2::scale_color_manual(values = c("#bababa","#000000"), 
+                                breaks = c("mdl_agesex","mdl_max_adj"), 
+                                labels = c("Age/sex adjustment","Extensive adjusted")) +
+    ggplot2::labs(x = "\nWeeks since COVID-19 diagnosis", y = "Hazard ratio and 95% confidence interval") +
+    ggplot2::theme_minimal() +
+    ggplot2::theme(panel.grid.major.x = ggplot2::element_blank(),
+                   panel.grid.minor = ggplot2::element_blank(),
+                   panel.spacing.x = ggplot2::unit(0.5, "lines"),
+                   panel.spacing.y = ggplot2::unit(0, "lines"),
+                   legend.key = ggplot2::element_rect(colour = NA, fill = NA),
+                   legend.title = ggplot2::element_blank(),
+                   legend.position="bottom",
+                   plot.background = ggplot2::element_rect(fill = "white", colour = "white")) +    
+    ggplot2::facet_wrap(cohort~., nrow = 1)
+  
+  ggplot2::ggsave(paste0("output/",filename,".png"), height = 210/2, width = 297, unit = "mm", dpi = 600, scale = 1)
```

Comment:
I would suggest to output this under the folder of OS-output - which is not linked to GitHub, though "output" is in gitignore and will not be pushed to GitHub.